Resources/Prototypes/Entities/Objects/Weapons/Guns/Ammunition/SpeedLoaders/rifle_light.yml

```
@@ -12,9 +12,9 @@
     mayTransfer: true
     whitelist:
       tags:
-        - CartridgeLightRifle
+        - CartridgeHeavyRifle #Sunrise-Edit 7.62x54
```

YAMLlint (1.37.1)

[warning] too few spaces before comment: expected 2

(comments)